web/js/yt2/youtube_video.js

```diff
@@ -1,31 +1,25 @@
 
 function playVideo1(){
```

Comment:
This function should not be repeated 3 times.

(Should have caught this one earlier)